### PR TITLE
Fix usb disk error_test4 failure by adding define_error = yes

### PR DESCRIPTION
**Pull request by chunfuwen**
Signed-off-by: chunfuwen <chwen@redhat.com>

# Format of PR title < sub-system: summary >
e.g
*virsh_migrate: Fix unsupported direct socket mode issue*

# Check lists by category
## If the PR is new cases
- [ ] Description of the cases
- [ ] Links of libvirt features, libvirt bugs or case IDs
- [ ] Test results

## If the PR is bug cases
- [ ] Bug descriptions or bug links
- [ ] Test results

## If the PR is a trivial fix
It it is the fix of typos, comments or documents, the description of PR could be skipped.

### CODE REVIEW

**Comment by chunfuwen** [comment]
Command 'virsh list' finished with 0 after 0.46665096282958984s
cleaning libvirtd logs...
DATA (filename=output.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stdout.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stderr.expected) => NOT FOUND (data sources: variant, test, file)
PASS 1-type_specific.io-github-autotest-libvirt.virtual_disks.multidisks.coldplug.single_disk_test.disk_bus_device_option.disk_bus_usb.device_disk.error_test4

**Comment by chunfuwen** [comment]
2021-04-05 08:06:15,514 stacktrace       L0045 ERROR| Traceback (most recent call last):
2021-04-05 08:06:15,514 stacktrace       L0045 ERROR|   File "/var/lib/avocado/data/avocado-vt/virttest/test-providers.d/downloads/io-github-autotest-libvirt/libvirt/tests/src/virtual_disks/virtual_disks_multidisks.py", line 1666, in run
2021-04-05 08:06:15,514 stacktrace       L0045 ERROR|     test.fail("Failed to define VM:\n%s" % xml_error)
2021-04-05 08:06:15,514 stacktrace       L0045 ERROR|   File "/var/tmp/libvirt-ci/runtest/avocado-vt/avocado/avocado/core/test.py", line 953, in fail
2021-04-05 08:06:15,514 stacktrace       L0045 ERROR|     raise exceptions.TestFail(message)
2021-04-05 08:06:15,514 stacktrace       L0045 ERROR| avocado.core.exceptions.TestFail: Failed to define VM:
2021-04-05 08:06:15,515 stacktrace       L0045 ERROR| Failed to define avocado-vt-vm1 for reason:
2021-04-05 08:06:15,515 stacktrace       L0045 ERROR| error: Failed to define domain from /tmp/xml_utils_temp_xnrj1j9m.xml
2021-04-05 08:06:15,515 stacktrace       L0045 ERROR| error: internal error: unexpected address type for usb disk

File: libvirt/tests/cfg/virtual_disks/virtual_disks_multidisks.cfg [hunk 1]
```diff
@@ -922,6 +922,7 @@
                                         - error_test4:
                                             no pseries
                                             status_error = "yes"
+                                            define_error = "yes"
```
Comment (Yingshun) [inline libvirt/tests/cfg/virtual_disks/virtual_disks_multidisks.cfg:925]: Does this error happen in all libvirt versions?

Comment (chunfuwen) [reply libvirt/tests/cfg/virtual_disks/virtual_disks_multidisks.cfg:925]: This is similar to error_test1 or 2. 
Whether it is start error or define error, the code can handle that, so no dependence to specific libvirt version

**Review by dzhengfy — APPROVED** [review]
LGTM.
Please help refine the PR description field to make it look clean before merged.

**Comment by chunfuwen** [comment]
> LGTM.
> Please help refine the PR description field to make it look clean before merged.

@dzhengfy 
As I observed, what git log can see is message embedded in https://github.com/autotest/tp-libvirt/pull/3419/commits
Conversions part message such as Format of PR title < sub-system: summary > ,which is brought by PR template WILL NOT be merged.

**Review by Yingshun — APPROVED** [review]

**Pull request merged**